woocommerce/woocommerce-ios
# Add initial set of screenshot mocks

This PR adds a WireMock setup for network mocking and an initial set of mocks to use with the WooCommerceScreenshots test. The mocks include faked data to make the screenshots look pretty.

To test:
* Run `rake mocks` to start the mock server.
* Run the test in `WooCommerceScreenshots` and confirm it passes with no failures or mocks errors. 
* Run `bundle exec fastlane screenshots` and confirm you get a set of pretty screenshots that use the faked data from the mocks.

Sample screenshots:

1|2|3|4|5
-|-|-|-|-
![iPhone 11 Pro Max-1-light-view-and-manage-orders](https://user-images.githubusercontent.com/8658164/74865877-b9043480-5349-11ea-8aef-b311969ceca4.png)|![iPhone 11 Pro Max-2-light-track-order-status](https://user-images.githubusercontent.com/8658164/74865881-bb668e80-5349-11ea-8108-e4c075e39e71.png)|![iPhone 11 Pro Max-3-light-look-up-specific-orders](https://user-images.githubusercontent.com/8658164/74865885-bbff2500-5349-11ea-89f8-08fc118202a9.png)|![iPhone 11 Pro Max-4-light-view-store-data](https://user-images.githubusercontent.com/8658164/74865888-bc97bb80-5349-11ea-9656-d5be4916352c.png)|![iPhone 11 Pro Max-5-light-get-notified-about-customer-reviews](https://user-images.githubusercontent.com/8658164/74865889-bd305200-5349-11ea-857b-99230ea24a44.png)


Update release notes:
